libs/game/sprite.ts

```diff
@@ -165,8 +165,8 @@ class Sprite extends sprites.BaseSprite {
     constructor(img: Image) {
         super(scene.SPRITE_Z);
 
-        this._x = Fx8(screen.width - img.width >> 1);
-        this._y = Fx8(screen.height - img.height >> 1);
+        this._x = Fx.zeroFx8;
```

Comment:
Not sure there is a good reason to break this; I realize it probably won't affect many people but why change it?

Reply:
Was just moving the initialization to the factories / `sprites.create` / `sprites.createProjectile` to keep them consistent there; good point though, no actual reason to break in case some person has extended this. Will put this back